lib/hammer_cli_katello/sync_plan.rb

```diff
@@ -9,6 +9,7 @@ class ListCommand < HammerCLIKatello::ListCommand
         field :sync_date, _("Start Date"), Fields::Date
         field :interval, _("Interval")
         field :enabled, _("Enabled"), Fields::Boolean
+        field :cron_expression, _("Cron Expression")
```

Comment:
There's no validation in Katello for this. i.e. require interval to be 'custom cron'

Reply:
Right..So my intent is to display the cron expression..Should we not do that?

Reply:
This hammer change is correct, but I feel we should add validation in Katello for this field, as it's only valid when `interval == 'custom cron'`